src/Microsoft.PowerShell.Commands.Management/commands/management/ResolvePathCommand.cs

```diff
@@ -103,6 +103,8 @@ protected override void ProcessRecord()
 
                     if (_relative)
                     {
+                        ReadOnlySpan<char> baseCache = null;
+                        ReadOnlySpan<char> adjustedBaseCache = null;
```

Comment:
Do we really need to init to null?

Reply:
Yes. I'm using them in a comparison before I'm assigning a new value.